src/scss/uswds-theme/_color.scss

```diff
@@ -149,50 +149,54 @@ State palette colors
-$theme-color-error-dark:       'site-darker-red';
-$theme-color-error-darker:     'site-darkest-red';
+$theme-color-error-lighter:    map-get($site-palette, 'site-lightest-red');
+$theme-color-error-light:      map-get($site-palette, 'site-lighter-red');
+$theme-color-error:            map-get($site-palette, 'site-red');
+$theme-color-error-dark:       map-get($site-palette, 'site-darker-red');
+$theme-color-error-darker:     map-get($site-palette, 'site-darkest-red');
 
 // Warning colors
 $theme-color-warning-family:   'yellow';
-$theme-color-warning-lighter:  'site-lightest-yellow';
-$theme-color-warning-light:    'site-lighter-yellow';
-$theme-color-warning:          'site-yellow';
-$theme-color-warning-dark:     'site-darker-yellow';
-$theme-color-warning-darker:   'site-darkest-yellow';
+$theme-color-warning-lighter:  map-get($site-palette, 'site-lightest-yellow');
+$theme-color-warning-light:    map-get($site-palette, 'site-lighter-yellow');
+$theme-color-warning:          map-get($site-palette, 'site-yellow');
+$theme-color-warning-dark:     map-get($site-palette, 'site-darker-yellow');
+$theme-color-warning-darker:   map-get($site-palette, 'site-darkest-yellow');
 
 // Success colors
 $theme-color-success-family:   'green';
-$theme-color-success-lighter:  'site-lightest-green';
-$theme-color-success-light:    'site-lighter-green';
-$theme-color-success:          'site-green';
-$theme-color-success-dark:     'site-darker-green';
-$theme-color-success-darker:   'site-darkest-green';
+$theme-color-success-lighter:  map-get($site-palette, 'site-lightest-green');
+$theme-color-success-light:    map-get($site-palette, 'site-lighter-green');
+$theme-color-success:          map-get($site-palette, 'site-green');
+$theme-color-success-dark:     map-get($site-palette, 'site-darker-green');
+$theme-color-success-darker:   map-get($site-palette, 'site-darkest-green');
 
 // Info colors
 $theme-color-info-family:      'teal';
-$theme-color-info-lighter:     'site-lightest-teal';
-$theme-color-info-light:       'site-lighter-teal';
-$theme-color-info:             'site-teal';
-$theme-color-info-dark:        'site-darker-teal';
-$theme-color-info-darker:      'site-darkest-teal';
+$theme-color-info-lighter:     map-get($site-palette, 'site-lightest-teal');
+$theme-color-info-light:       map-get($site-palette, 'site-lighter-teal');
+$theme-color-info:             map-get($site-palette, 'site-teal');
+$theme-color-info-dark:        map-get($site-palette, 'site-darker-teal');
+$theme-color-info-darker:      map-get($site-palette, 'site-darkest-teal');
 
 // Disabled colors
 $theme-color-disabled-family:  'grey';
-$theme-color-disabled-light:   'site-lightest-grey';
-$theme-color-disabled:         'site-disabled-grey';
-$theme-color-disabled-dark:    'site-darker-grey';
+// TODO: Revert back to `map-get` once https://github.com/uswds/uswds/pull/3762 is released.
+$theme-color-disabled-light:   'gray-5'; // map-get($site-palette, 'site-lightest-grey');
```

Comment:
should we standardize on `gray` vs `grey`? (looking at `$theme-color-disabled-family` above). My inclination is to use `gray` because that's standard for US English?

Reply:
> should we standardize on `gray` vs `grey`? (looking at `$theme-color-disabled-family` above). My inclination is to use `gray` because that's standard for US English?

tl;dr: I think so, yes.

It might be worth getting broader UX feedback here, but it kinda seems like the whole set of `$theme-color-grey` variables should be eliminated altogether. At least comparing to the [base USWDS theme color tokens](https://designsystem.digital.gov/design-tokens/color/theme-tokens/#customizing-theme-color-tokens), I don't see any reference to where grey originates from. Grays appear to fit within the "base" family, and we only reference grey under the base colors in [our own color guide](https://design.login.gov/color/).

It may make sense to keep our own `$site-palette`, but considering that USWDS [spells it as `gray`](https://github.com/uswds/uswds/blob/92ae6f8d3d4b8f7c25a246ca03daed9194846511/src/stylesheets/core/_system-tokens.scss#L497-L499) in their own system tokens, I agree it'd make sense to align it.

In the short-term though, I'm a bit wary to include it here, since it has some cascading effects, but in terms of making breaking changes in one pass, it could be wise to do before publishing a version of this package that contains USWDS 2.9.0.